themes/tiddlywiki/vanilla/base.tid

```diff
@@ -479,7 +479,7 @@ button svg.tc-image-button, button .tc-image-button img {
 }
 
 @media (max-width: {{$:/themes/tiddlywiki/vanilla/metrics/sidebarbreakpoint}}) {
-	
```

Comment:
My editor also cleaned up some trailing whitespace characters, hope you don't mind.